python/test/unit/language/test_matmul.py

```diff
@@ -316,6 +316,8 @@ def fp8e8m0_to_float32(scale):
 @pytest.mark.parametrize("NUM_STAGES", [1, 3])
 @pytest.mark.parametrize("NUM_WARPS", [4, 8])
 @pytest.mark.parametrize("nonKDim", ([0, 16, 32] if is_hip_cdna() else [0]))
+@pytest.mark.skipif(is_xpu() and not torch.xpu.get_device_capability()['has_subgroup_matrix_multiply_accumulate'],
```

Comment:
Just for the case when `has_subgroup_matrix_multiply_accumulate` not in capabilities.
```suggestion
@pytest.mark.skipif(is_xpu() and not torch.xpu.get_device_capability().get('has_subgroup_matrix_multiply_accumulate', False),
```
In general, this decorator is executed in runtime, what is not convenient and Python best practice is minimize import-time logic. IMHO it is better to move this conditional skip the the test body.